cluster/gce/manifests/etcd.manifest

```diff
@@ -21,6 +21,7 @@
         "cpu": {{ cpulimit }}
       }
     },
+    "_comment": "If etcd name defined by --name is modified, please make sure ETCD_NAME in migrate-if-needed.sh is changed to the same: https://github.com/kubernetes/kubernetes/blob/v1.19.0-beta.2/cluster/images/etcd/migrate-if-needed.sh#L49.",
```

Comment:
I really don't like adding a file to the pod. Let's make that a file comment.

You can do that:
https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/manifests/kube-addon-manager.yaml#L25

You just can't do that within the Command field (which is []string: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/types.go#L2136) as you did before.

Reply:
As discussed with @wojtek-t  offline, etcd.manifest is json file, which does not support comment. We will remove the comment from here for now.